### ISSUE TITLE
read method steps from excel

### ISSUE THREAD

**Issue by schumannj** [budschi/nomad-synthesis-plugin#2]
some excel exports contain information about the steps of the preparation. create a function that reads the excel sheet 'Recipe'

**Comment by schumannj** [comment]
because of missing permissions I forked the project here
https://github.com/schumannj/nomad-synthesis-plugin/tree/branch_julia

**Comment by budschi** [comment]
Oh, I could also adjust the rights. Even better would be if we move the general stuff to the nomad materials processing plugin

**Comment by schumannj** [comment]
Yes, that would be best to move it into the materials processing plugin.
